[2024-02-08T00:08:26.139+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: lighthouse_challenge.extracts.teste scheduled__2024-02-07T00:00:00+00:00 [queued]>
[2024-02-08T00:08:26.164+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: lighthouse_challenge.extracts.teste scheduled__2024-02-07T00:00:00+00:00 [queued]>
[2024-02-08T00:08:26.167+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-08T00:08:26.648+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): extracts.teste> on 2024-02-07 00:00:00+00:00
[2024-02-08T00:08:26.738+0000] {standard_task_runner.py:60} INFO - Started process 71 to run task
[2024-02-08T00:08:26.822+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'lighthouse_challenge', 'extracts.teste', 'scheduled__2024-02-07T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmplp4ss0ic']
[2024-02-08T00:08:26.865+0000] {standard_task_runner.py:88} INFO - Job 31: Subtask extracts.teste
[2024-02-08T00:08:28.023+0000] {task_command.py:423} INFO - Running <TaskInstance: lighthouse_challenge.extracts.teste scheduled__2024-02-07T00:00:00+00:00 [running]> on host 9f3366cf5953
[2024-02-08T00:08:28.366+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='lighthouse_challenge' AIRFLOW_CTX_TASK_ID='extracts.teste' AIRFLOW_CTX_EXECUTION_DATE='2024-02-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-07T00:00:00+00:00'
[2024-02-08T00:08:28.382+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-02-08T00:08:28.389+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'pwd']
[2024-02-08T00:08:28.448+0000] {subprocess.py:86} INFO - Output:
[2024-02-08T00:08:28.529+0000] {subprocess.py:93} INFO - /tmp/***tmpcch7244p
[2024-02-08T00:08:28.547+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-02-08T00:08:28.660+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=lighthouse_challenge, task_id=extracts.teste, execution_date=20240207T000000, start_date=20240208T000826, end_date=20240208T000828
[2024-02-08T00:08:28.775+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-08T00:08:28.886+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-08T01:03:34.723+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: lighthouse_challenge.extracts.teste scheduled__2024-02-07T00:00:00+00:00 [queued]>
[2024-02-08T01:03:34.730+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: lighthouse_challenge.extracts.teste scheduled__2024-02-07T00:00:00+00:00 [queued]>
[2024-02-08T01:03:34.730+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-08T01:03:34.741+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): extracts.teste> on 2024-02-07 00:00:00+00:00
[2024-02-08T01:03:34.770+0000] {standard_task_runner.py:60} INFO - Started process 120 to run task
[2024-02-08T01:03:34.775+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'lighthouse_challenge', 'extracts.teste', 'scheduled__2024-02-07T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp8dehsw00']
[2024-02-08T01:03:34.809+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask extracts.teste
[2024-02-08T01:03:35.227+0000] {task_command.py:423} INFO - Running <TaskInstance: lighthouse_challenge.extracts.teste scheduled__2024-02-07T00:00:00+00:00 [running]> on host adad592ee517
[2024-02-08T01:03:35.383+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='lighthouse_challenge' AIRFLOW_CTX_TASK_ID='extracts.teste' AIRFLOW_CTX_EXECUTION_DATE='2024-02-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-07T00:00:00+00:00'
[2024-02-08T01:03:35.415+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-02-08T01:03:35.424+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/meltano && pip install meltano && meltano install && meltano run extract-csv']
[2024-02-08T01:03:35.475+0000] {subprocess.py:86} INFO - Output:
[2024-02-08T01:03:37.569+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2024-02-08T01:03:39.596+0000] {subprocess.py:93} INFO - Collecting meltano
[2024-02-08T01:03:39.763+0000] {subprocess.py:93} INFO -   Downloading meltano-3.3.1-py3-none-any.whl.metadata (9.0 kB)
[2024-02-08T01:03:39.913+0000] {subprocess.py:93} INFO - Collecting aiodocker<0.22.0,>=0.21.0 (from meltano)
[2024-02-08T01:03:39.947+0000] {subprocess.py:93} INFO -   Downloading aiodocker-0.21.0-py3-none-any.whl (34 kB)
[2024-02-08T01:03:39.981+0000] {subprocess.py:93} INFO - Requirement already satisfied: alembic<2.0.0,>=1.13.1 in /home/***/.local/lib/python3.8/site-packages (from meltano) (1.13.1)
[2024-02-08T01:03:40.117+0000] {subprocess.py:93} INFO - Collecting atomicwrites<2.0.0,>=1.2.1 (from meltano)
[2024-02-08T01:03:40.146+0000] {subprocess.py:93} INFO -   Downloading atomicwrites-1.4.1.tar.gz (14 kB)
[2024-02-08T01:03:40.200+0000] {subprocess.py:93} INFO -   Preparing metadata (setup.py): started
[2024-02-08T01:03:40.941+0000] {subprocess.py:93} INFO -   Preparing metadata (setup.py): finished with status 'done'
[2024-02-08T01:03:41.055+0000] {subprocess.py:93} INFO - Collecting check-jsonschema<0.23.0,>=0.22.0 (from meltano)
[2024-02-08T01:03:41.088+0000] {subprocess.py:93} INFO -   Downloading check_jsonschema-0.22.0-py3-none-any.whl (196 kB)
[2024-02-08T01:03:41.187+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 196.3/196.3 kB 2.2 MB/s eta 0:00:00
[2024-02-08T01:03:41.189+0000] {subprocess.py:93} INFO - Requirement already satisfied: click<9.0,>=8.1 in /home/***/.local/lib/python3.8/site-packages (from meltano) (8.1.7)
[2024-02-08T01:03:41.251+0000] {subprocess.py:93} INFO - Collecting click-default-group<2.0.0,>=1.2.4 (from meltano)
[2024-02-08T01:03:41.293+0000] {subprocess.py:93} INFO -   Downloading click_default_group-1.2.4-py2.py3-none-any.whl.metadata (2.8 kB)
[2024-02-08T01:03:41.313+0000] {subprocess.py:93} INFO - Requirement already satisfied: click-didyoumean<0.4.0,>=0.3.0 in /home/***/.local/lib/python3.8/site-packages (from meltano) (0.3.0)
[2024-02-08T01:03:41.314+0000] {subprocess.py:93} INFO - Requirement already satisfied: croniter<3.0.0,>=2.0.1 in /home/***/.local/lib/python3.8/site-packages (from meltano) (2.0.1)
[2024-02-08T01:03:41.399+0000] {subprocess.py:93} INFO - Collecting fasteners<0.20,>=0.19 (from meltano)
[2024-02-08T01:03:41.463+0000] {subprocess.py:93} INFO -   Downloading fasteners-0.19-py3-none-any.whl.metadata (4.9 kB)
[2024-02-08T01:03:41.642+0000] {subprocess.py:93} INFO - Collecting flatten-dict<1,>=0 (from meltano)
[2024-02-08T01:03:41.683+0000] {subprocess.py:93} INFO -   Downloading flatten_dict-0.4.2-py2.py3-none-any.whl (9.7 kB)
[2024-02-08T01:03:41.862+0000] {subprocess.py:93} INFO - Collecting importlib-resources<7.0.0,>=6.1.1 (from meltano)
[2024-02-08T01:03:41.898+0000] {subprocess.py:93} INFO -   Downloading importlib_resources-6.1.1-py3-none-any.whl.metadata (4.1 kB)
[2024-02-08T01:03:41.949+0000] {subprocess.py:93} INFO - Requirement already satisfied: jinja2<4.0.0,>=3.1.3 in /home/***/.local/lib/python3.8/site-packages (from meltano) (3.1.3)
[2024-02-08T01:03:41.950+0000] {subprocess.py:93} INFO - Requirement already satisfied: jsonschema<5.0,>=4.20 in /home/***/.local/lib/python3.8/site-packages (from meltano) (4.20.0)
[2024-02-08T01:03:41.958+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging<24.0,>=23.2 in /home/***/.local/lib/python3.8/site-packages (from meltano) (23.2)
[2024-02-08T01:03:42.157+0000] {subprocess.py:93} INFO - Collecting platformdirs<5.0.0,>=4.1.0 (from meltano)
[2024-02-08T01:03:42.194+0000] {subprocess.py:93} INFO -   Downloading platformdirs-4.2.0-py3-none-any.whl.metadata (11 kB)
[2024-02-08T01:03:42.234+0000] {subprocess.py:93} INFO - Requirement already satisfied: psutil<6.0.0,>=5.9.7 in /home/***/.local/lib/python3.8/site-packages (from meltano) (5.9.7)
[2024-02-08T01:03:42.239+0000] {subprocess.py:93} INFO - Requirement already satisfied: pyjwt<3.0.0,>=2.6.0 in /home/***/.local/lib/python3.8/site-packages (from meltano) (2.8.0)
[2024-02-08T01:03:42.248+0000] {subprocess.py:93} INFO - Requirement already satisfied: python-dateutil<3.0.0,>=2.8.2 in /home/***/.local/lib/python3.8/site-packages (from meltano) (2.8.2)
[2024-02-08T01:03:42.250+0000] {subprocess.py:93} INFO - Requirement already satisfied: python-dotenv<2.0.0,>=1.0.0 in /home/***/.local/lib/python3.8/site-packages (from meltano) (1.0.0)
[2024-02-08T01:03:42.252+0000] {subprocess.py:93} INFO - Requirement already satisfied: python-slugify<9.0.0,>=8.0.1 in /home/***/.local/lib/python3.8/site-packages (from meltano) (8.0.1)
[2024-02-08T01:03:42.322+0000] {subprocess.py:93} INFO - Collecting python-ulid<2.0.0,>=1.1.0 (from meltano)
[2024-02-08T01:03:42.363+0000] {subprocess.py:93} INFO -   Downloading python_ulid-1.1.0-py3-none-any.whl (9.4 kB)
[2024-02-08T01:03:42.402+0000] {subprocess.py:93} INFO - Requirement already satisfied: pyyaml<7.0.0,>=6.0.1 in /home/***/.local/lib/python3.8/site-packages (from meltano) (6.0.1)
[2024-02-08T01:03:42.490+0000] {subprocess.py:93} INFO - Collecting questionary<3.0.0,>=2.0.1 (from meltano)
[2024-02-08T01:03:42.523+0000] {subprocess.py:93} INFO -   Downloading questionary-2.0.1-py3-none-any.whl.metadata (5.4 kB)
[2024-02-08T01:03:42.559+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests<3.0.0,>=2.31.0 in /home/***/.local/lib/python3.8/site-packages (from meltano) (2.31.0)
[2024-02-08T01:03:42.560+0000] {subprocess.py:93} INFO - Requirement already satisfied: rich<14.0.0,>=13.7.0 in /home/***/.local/lib/python3.8/site-packages (from meltano) (13.7.0)
[2024-02-08T01:03:42.930+0000] {subprocess.py:93} INFO - Collecting ruamel.yaml<0.18.0,>=0.17.21 (from meltano)
[2024-02-08T01:03:42.966+0000] {subprocess.py:93} INFO -   Downloading ruamel.yaml-0.17.40-py3-none-any.whl.metadata (19 kB)
[2024-02-08T01:03:43.131+0000] {subprocess.py:93} INFO - Collecting smart-open<7.0.0,>=6.4.0 (from meltano)
[2024-02-08T01:03:43.178+0000] {subprocess.py:93} INFO -   Downloading smart_open-6.4.0-py3-none-any.whl.metadata (21 kB)
[2024-02-08T01:03:43.315+0000] {subprocess.py:93} INFO - Collecting snowplow-tracker<2.0.0,>=1.0.1 (from meltano)
[2024-02-08T01:03:43.354+0000] {subprocess.py:93} INFO -   Downloading snowplow_tracker-1.0.1-py3-none-any.whl.metadata (5.6 kB)
[2024-02-08T01:03:43.952+0000] {subprocess.py:93} INFO - Collecting sqlalchemy<3.0.0,>=2.0.25 (from meltano)
[2024-02-08T01:03:43.995+0000] {subprocess.py:93} INFO -   Downloading SQLAlchemy-2.0.25-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (9.6 kB)
[2024-02-08T01:03:44.165+0000] {subprocess.py:93} INFO - Collecting structlog<25.0.0,>=24.1.0 (from meltano)
[2024-02-08T01:03:44.206+0000] {subprocess.py:93} INFO -   Downloading structlog-24.1.0-py3-none-any.whl.metadata (6.9 kB)
[2024-02-08T01:03:44.282+0000] {subprocess.py:93} INFO - Requirement already satisfied: tabulate<0.10.0,>=0.9.0 in /home/***/.local/lib/python3.8/site-packages (from meltano) (0.9.0)
[2024-02-08T01:03:44.285+0000] {subprocess.py:93} INFO - Requirement already satisfied: typing-extensions<5.0.0,>=4.9.0 in /home/***/.local/lib/python3.8/site-packages (from meltano) (4.9.0)
[2024-02-08T01:03:44.420+0000] {subprocess.py:93} INFO - Collecting tzlocal<6.0,>=5.2 (from meltano)
[2024-02-08T01:03:44.452+0000] {subprocess.py:93} INFO -   Downloading tzlocal-5.2-py3-none-any.whl.metadata (7.8 kB)
[2024-02-08T01:03:44.515+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<2 in /home/***/.local/lib/python3.8/site-packages (from meltano) (1.26.18)
[2024-02-08T01:03:44.521+0000] {subprocess.py:93} INFO - Requirement already satisfied: virtualenv<21.0.0,>=20.25.0 in /home/***/.local/lib/python3.8/site-packages (from meltano) (20.25.0)
[2024-02-08T01:03:44.644+0000] {subprocess.py:93} INFO - Collecting yaspin<3.0.0,>=2.3.0 (from meltano)
[2024-02-08T01:03:44.680+0000] {subprocess.py:93} INFO -   Downloading yaspin-2.5.0-py3-none-any.whl.metadata (14 kB)
[2024-02-08T01:03:44.779+0000] {subprocess.py:93} INFO - Requirement already satisfied: aiohttp>=3.6 in /home/***/.local/lib/python3.8/site-packages (from aiodocker<0.22.0,>=0.21.0->meltano) (3.9.1)
[2024-02-08T01:03:44.825+0000] {subprocess.py:93} INFO - Requirement already satisfied: Mako in /home/***/.local/lib/python3.8/site-packages (from alembic<2.0.0,>=1.13.1->meltano) (1.3.0)
[2024-02-08T01:03:44.830+0000] {subprocess.py:93} INFO - Requirement already satisfied: importlib-metadata in /home/***/.local/lib/python3.8/site-packages (from alembic<2.0.0,>=1.13.1->meltano) (6.11.0)
[2024-02-08T01:03:44.882+0000] {subprocess.py:93} INFO - Collecting ruamel.yaml<0.18.0,>=0.17.21 (from meltano)
[2024-02-08T01:03:44.943+0000] {subprocess.py:93} INFO -   Downloading ruamel.yaml-0.17.21-py3-none-any.whl (109 kB)
[2024-02-08T01:03:45.011+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 109.5/109.5 kB 1.8 MB/s eta 0:00:00
[2024-02-08T01:03:45.139+0000] {subprocess.py:93} INFO - Collecting ruamel.yaml.clib>=0.2.6 (from ruamel.yaml<0.18.0,>=0.17.21->meltano)
[2024-02-08T01:03:45.171+0000] {subprocess.py:93} INFO -   Downloading ruamel.yaml.clib-0.2.8-cp38-cp38-manylinux_2_24_aarch64.whl.metadata (2.2 kB)
[2024-02-08T01:03:45.231+0000] {subprocess.py:93} INFO - Requirement already satisfied: pytz>2021.1 in /home/***/.local/lib/python3.8/site-packages (from croniter<3.0.0,>=2.0.1->meltano) (2023.3.post1)
[2024-02-08T01:03:45.239+0000] {subprocess.py:93} INFO - Requirement already satisfied: six<2.0,>=1.12 in /home/***/.local/lib/python3.8/site-packages (from flatten-dict<1,>=0->meltano) (1.16.0)
[2024-02-08T01:03:45.253+0000] {subprocess.py:93} INFO - Requirement already satisfied: zipp>=3.1.0 in /home/***/.local/lib/python3.8/site-packages (from importlib-resources<7.0.0,>=6.1.1->meltano) (3.17.0)
[2024-02-08T01:03:45.259+0000] {subprocess.py:93} INFO - Requirement already satisfied: MarkupSafe>=2.0 in /home/***/.local/lib/python3.8/site-packages (from jinja2<4.0.0,>=3.1.3->meltano) (2.1.3)
[2024-02-08T01:03:45.292+0000] {subprocess.py:93} INFO - Requirement already satisfied: attrs>=22.2.0 in /home/***/.local/lib/python3.8/site-packages (from jsonschema<5.0,>=4.20->meltano) (23.2.0)
[2024-02-08T01:03:45.300+0000] {subprocess.py:93} INFO - Requirement already satisfied: jsonschema-specifications>=2023.03.6 in /home/***/.local/lib/python3.8/site-packages (from jsonschema<5.0,>=4.20->meltano) (2023.12.1)
[2024-02-08T01:03:45.303+0000] {subprocess.py:93} INFO - Requirement already satisfied: pkgutil-resolve-name>=1.3.10 in /home/***/.local/lib/python3.8/site-packages (from jsonschema<5.0,>=4.20->meltano) (1.3.10)
[2024-02-08T01:03:45.306+0000] {subprocess.py:93} INFO - Requirement already satisfied: referencing>=0.28.4 in /home/***/.local/lib/python3.8/site-packages (from jsonschema<5.0,>=4.20->meltano) (0.32.1)
[2024-02-08T01:03:45.309+0000] {subprocess.py:93} INFO - Requirement already satisfied: rpds-py>=0.7.1 in /home/***/.local/lib/python3.8/site-packages (from jsonschema<5.0,>=4.20->meltano) (0.17.1)
[2024-02-08T01:03:45.374+0000] {subprocess.py:93} INFO - Requirement already satisfied: text-unidecode>=1.3 in /home/***/.local/lib/python3.8/site-packages (from python-slugify<9.0.0,>=8.0.1->meltano) (1.3)
[2024-02-08T01:03:45.506+0000] {subprocess.py:93} INFO - Collecting prompt_toolkit<=3.0.36,>=2.0 (from questionary<3.0.0,>=2.0.1->meltano)
[2024-02-08T01:03:47.045+0000] {subprocess.py:93} INFO -   Downloading prompt_toolkit-3.0.36-py3-none-any.whl (386 kB)
[2024-02-08T01:03:47.137+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 386.4/386.4 kB 236.2 kB/s eta 0:00:00
[2024-02-08T01:03:47.180+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests<3.0.0,>=2.31.0->meltano) (3.3.2)
[2024-02-08T01:03:47.183+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests<3.0.0,>=2.31.0->meltano) (3.6)
[2024-02-08T01:03:47.183+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests<3.0.0,>=2.31.0->meltano) (2023.11.17)
[2024-02-08T01:03:47.198+0000] {subprocess.py:93} INFO - Requirement already satisfied: markdown-it-py>=2.2.0 in /home/***/.local/lib/python3.8/site-packages (from rich<14.0.0,>=13.7.0->meltano) (3.0.0)
[2024-02-08T01:03:47.203+0000] {subprocess.py:93} INFO - Requirement already satisfied: pygments<3.0.0,>=2.13.0 in /home/***/.local/lib/python3.8/site-packages (from rich<14.0.0,>=13.7.0->meltano) (2.17.2)
[2024-02-08T01:03:47.369+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from sqlalchemy<3.0.0,>=2.0.25->meltano) (3.0.3)
[2024-02-08T01:03:47.403+0000] {subprocess.py:93} INFO - Requirement already satisfied: backports.zoneinfo in /home/***/.local/lib/python3.8/site-packages (from tzlocal<6.0,>=5.2->meltano) (0.2.1)
[2024-02-08T01:03:47.442+0000] {subprocess.py:93} INFO - Requirement already satisfied: distlib<1,>=0.3.7 in /home/***/.local/lib/python3.8/site-packages (from virtualenv<21.0.0,>=20.25.0->meltano) (0.3.8)
[2024-02-08T01:03:47.443+0000] {subprocess.py:93} INFO - Requirement already satisfied: filelock<4,>=3.12.2 in /home/***/.local/lib/python3.8/site-packages (from virtualenv<21.0.0,>=20.25.0->meltano) (3.13.1)
[2024-02-08T01:03:47.450+0000] {subprocess.py:93} INFO - Requirement already satisfied: termcolor<3.0,>=2.3 in /home/***/.local/lib/python3.8/site-packages (from yaspin<3.0.0,>=2.3.0->meltano) (2.4.0)
[2024-02-08T01:03:47.464+0000] {subprocess.py:93} INFO - Requirement already satisfied: multidict<7.0,>=4.5 in /home/***/.local/lib/python3.8/site-packages (from aiohttp>=3.6->aiodocker<0.22.0,>=0.21.0->meltano) (6.0.4)
[2024-02-08T01:03:47.465+0000] {subprocess.py:93} INFO - Requirement already satisfied: yarl<2.0,>=1.0 in /home/***/.local/lib/python3.8/site-packages (from aiohttp>=3.6->aiodocker<0.22.0,>=0.21.0->meltano) (1.9.4)
[2024-02-08T01:03:47.466+0000] {subprocess.py:93} INFO - Requirement already satisfied: frozenlist>=1.1.1 in /home/***/.local/lib/python3.8/site-packages (from aiohttp>=3.6->aiodocker<0.22.0,>=0.21.0->meltano) (1.4.1)
[2024-02-08T01:03:47.466+0000] {subprocess.py:93} INFO - Requirement already satisfied: aiosignal>=1.1.2 in /home/***/.local/lib/python3.8/site-packages (from aiohttp>=3.6->aiodocker<0.22.0,>=0.21.0->meltano) (1.3.1)
[2024-02-08T01:03:47.466+0000] {subprocess.py:93} INFO - Requirement already satisfied: async-timeout<5.0,>=4.0 in /home/***/.local/lib/python3.8/site-packages (from aiohttp>=3.6->aiodocker<0.22.0,>=0.21.0->meltano) (4.0.3)
[2024-02-08T01:03:47.541+0000] {subprocess.py:93} INFO - Requirement already satisfied: mdurl~=0.1 in /home/***/.local/lib/python3.8/site-packages (from markdown-it-py>=2.2.0->rich<14.0.0,>=13.7.0->meltano) (0.1.2)
[2024-02-08T01:03:47.554+0000] {subprocess.py:93} INFO - Requirement already satisfied: wcwidth in /home/***/.local/lib/python3.8/site-packages (from prompt_toolkit<=3.0.36,>=2.0->questionary<3.0.0,>=2.0.1->meltano) (0.2.13)
[2024-02-08T01:03:47.723+0000] {subprocess.py:93} INFO - Downloading meltano-3.3.1-py3-none-any.whl (377 kB)
[2024-02-08T01:03:47.828+0000] {subprocess.py:93} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 377.5/377.5 kB 3.6 MB/s eta 0:00:00
[2024-02-08T01:03:47.858+0000] {subprocess.py:93} INFO - Downloading click_default_group-1.2.4-py2.py3-none-any.whl (4.1 kB)
[2024-02-08T01:03:47.897+0000] {subprocess.py:93} INFO - Downloading fasteners-0.19-py3-none-any.whl (18 kB)
[2024-02-08T01:03:47.943+0000] {subprocess.py:93} INFO - Downloading importlib_resources-6.1.1-py3-none-any.whl (33 kB)
[2024-02-08T01:03:47.983+0000] {subprocess.py:93} INFO - Downloading platformdirs-4.2.0-py3-none-any.whl (17 kB)
[2024-02-08T01:03:48.070+0000] {subprocess.py:93} INFO - Downloading questionary-2.0.1-py3-none-any.whl (34 kB)
[2024-02-08T01:03:48.120+0000] {subprocess.py:93} INFO - Downloading smart_open-6.4.0-py3-none-any.whl (57 kB)
[2024-02-08T01:03:48.138+0000] {subprocess.py:93} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 57.0/57.0 kB 3.5 MB/s eta 0:00:00
[2024-02-08T01:03:48.172+0000] {subprocess.py:93} INFO - Downloading snowplow_tracker-1.0.1-py3-none-any.whl (43 kB)
[2024-02-08T01:03:48.193+0000] {subprocess.py:93} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 43.4/43.4 kB 3.2 MB/s eta 0:00:00
[2024-02-08T01:03:48.226+0000] {subprocess.py:93} INFO - Downloading SQLAlchemy-2.0.25-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (3.1 MB)
[2024-02-08T01:03:48.911+0000] {subprocess.py:93} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.1/3.1 MB 4.5 MB/s eta 0:00:00
[2024-02-08T01:03:48.946+0000] {subprocess.py:93} INFO - Downloading structlog-24.1.0-py3-none-any.whl (65 kB)
[2024-02-08T01:03:48.966+0000] {subprocess.py:93} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 65.7/65.7 kB 5.5 MB/s eta 0:00:00
[2024-02-08T01:03:48.993+0000] {subprocess.py:93} INFO - Downloading tzlocal-5.2-py3-none-any.whl (17 kB)
[2024-02-08T01:03:49.032+0000] {subprocess.py:93} INFO - Downloading yaspin-2.5.0-py3-none-any.whl (18 kB)
[2024-02-08T01:03:49.087+0000] {subprocess.py:93} INFO - Downloading ruamel.yaml.clib-0.2.8-cp38-cp38-manylinux_2_24_aarch64.whl (641 kB)
[2024-02-08T01:03:49.132+0000] {subprocess.py:93} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 641.5/641.5 kB 15.4 MB/s eta 0:00:00
[2024-02-08T01:03:49.162+0000] {subprocess.py:93} INFO - Building wheels for collected packages: atomicwrites
[2024-02-08T01:03:49.165+0000] {subprocess.py:93} INFO -   Building wheel for atomicwrites (setup.py): started
[2024-02-08T01:03:54.939+0000] {subprocess.py:93} INFO -   Building wheel for atomicwrites (setup.py): finished with status 'done'
[2024-02-08T01:03:54.954+0000] {subprocess.py:93} INFO -   Created wheel for atomicwrites: filename=atomicwrites-1.4.1-py2.py3-none-any.whl size=6942 sha256=b9a3014bf3e0ace9ea2bad532720ccdf6a7fbc5c6bc2bff6abd8c883c90fcae9
[2024-02-08T01:03:54.956+0000] {subprocess.py:93} INFO -   Stored in directory: /home/***/.cache/pip/wheels/ce/24/c1/56b1ca77dda9577b75eb88d7ec50425f103ec2f6170c6e97df
[2024-02-08T01:03:54.967+0000] {subprocess.py:93} INFO - Successfully built atomicwrites
[2024-02-08T01:04:03.779+0000] {subprocess.py:93} INFO - Installing collected packages: yaspin, tzlocal, structlog, sqlalchemy, smart-open, ruamel.yaml.clib, python-ulid, prompt_toolkit, platformdirs, importlib-resources, flatten-dict, fasteners, click-default-group, atomicwrites, snowplow-tracker, ruamel.yaml, questionary, aiodocker, check-jsonschema, meltano
[2024-02-08T01:04:04.163+0000] {subprocess.py:93} INFO -   Attempting uninstall: sqlalchemy
[2024-02-08T01:04:04.173+0000] {subprocess.py:93} INFO -     Found existing installation: SQLAlchemy 1.4.51
[2024-02-08T01:04:04.195+0000] {subprocess.py:93} INFO -     Uninstalling SQLAlchemy-1.4.51:
[2024-02-08T01:04:04.218+0000] {subprocess.py:93} INFO - ERROR: Could not install packages due to an OSError: [Errno 2] No such file or directory: 'LICENSE'
[2024-02-08T01:04:04.219+0000] {subprocess.py:93} INFO - 
[2024-02-08T01:04:04.269+0000] {subprocess.py:93} INFO - 
[2024-02-08T01:04:04.270+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.3.2 -> 24.0
[2024-02-08T01:04:04.270+0000] {subprocess.py:93} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2024-02-08T01:04:04.826+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-02-08T01:04:06.165+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-02-08T01:04:07.391+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=lighthouse_challenge, task_id=extracts.teste, execution_date=20240207T000000, start_date=20240208T010334, end_date=20240208T010407
[2024-02-08T01:04:09.112+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 3 for task extracts.teste (Bash command failed. The command returned a non-zero exit code 1.; 120)
[2024-02-08T01:04:09.536+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-08T01:04:09.884+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
